kubernetes/nodejs-helm/waypoint.hcl

```diff
@@ -21,13 +21,14 @@ app "nodejs-helm-web" {
     use "helm" {
       name  = app.name
       chart = "${path.app}/helm"
-
+      namespace = "dev"
```

Comment:
Was this change intended?

Reply:
Nope! 😄 I moved this PR back to draft after I saw that, need to undo this, as well as the changes under static-application-configuration as well as nomad.

Reply:
Cool cool, no worries! 👍🏻